src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -270,8 +275,12 @@ def __await__(self) -> ResponseType:
             if self._cython_call.is_locally_cancelled():
                 raise asyncio.CancelledError()
             else:
+                call_status = self._cython_call._status
+                debug_error_string = None
+                if call_status is not None:
+                    debug_error_string = call_status._debug_error_string
                 raise _create_rpc_error(self._cython_call._initial_metadata,
-                                        self._cython_call._status)
+                                        call_status, debug_error_string)
```

Comment:
by default, I would say that will return an empty string when it is not available. does the Optional make sense for that function? is this function used in other places where there is no `debug_error_string`? If so, would it make sense on turning an empty sting as None value?

Reply:
The previous value was
```
  debug_error_string = "None"
```
I suppose empty string makes sense, and maybe the "None" wasn't intentional...